### PR TITLE
[IOTDB-924] Support insert multi rows

### PR DESCRIPTION
**Pull request by Superainbower**
## Description
JIRA issue: https://issues.apache.org/jira/projects/IOTDB/issues/IOTDB-924

IoTDB only support insert one row in the insert sentence now, we need to support insert multi rows
image

### Content1 ...
modify The SqlBase.g4 flie like this
insertValuesSpec
    :(COMMA? insertMultiValue)*
    ;
and add 
insertMultiValue
    : LR_BRACKET dateFormat (COMMA constant)+ RR_BRACKET
    | LR_BRACKET INT (COMMA constant)+ RR_BRACKET
    ;
### Content2 ...
modify the type of time in InsertOperator.java and IoTDBSqlVisitor.java to support save multi rows(values & times)

### Content3 ...
new InsertRowsPlan in PhysicalGenerator.java to support insert multi rows

<hr>

This PR has:
- [ ] been self-reviewed.
    - [ ] concurrent read
    - [ ] concurrent write
    - [ ] concurrent read and write 
- [x] added documentation for new or modified features or behaviors.
- [ ] added Javadocs for most classes and all non-trivial methods. 
- [ ] added or updated version, __license__, or notice information
- [x] added comments explaining the "why" and the intent of the code wherever would not be obvious 
  for an unfamiliar reader.
- [ ] added unit tests or modified existing tests to cover new code paths, ensuring the threshold 
  for code coverage.
- [x] added integration tests.
- [x] been tested in a test IoTDB cluster.

<hr>

### CODE REVIEW

**Review by HTHou — APPROVED** [review]
LGTM

File: server/src/main/java/org/apache/iotdb/db/qp/logical/crud/InsertOperator.java [hunk 1]
```diff
@@ -46,11 +46,11 @@ public void setValueList(String[] insertValue) {
     this.valueList = insertValue;
   }
 
-  public long getTime() {
+  public long[] getTime() {
```
Comment (qiaojialin) [inline server/src/main/java/org/apache/iotdb/db/qp/logical/crud/InsertOperator.java:49]:
```suggestion
  public long[] getTimes() {
```

File: server/src/main/java/org/apache/iotdb/db/qp/logical/crud/InsertOperator.java [hunk 2]
```diff
@@ -46,11 +46,11 @@ public void setValueList(String[] insertValue) {
...
     return time;
   }
 
-  public void setTime(long time) {
+  public void setTime(long[] time) {
```
Comment (qiaojialin) [inline server/src/main/java/org/apache/iotdb/db/qp/logical/crud/InsertOperator.java:53]:
```suggestion
  public void setTimes(long[] times) {
```

Comment (Superainbower) [reply server/src/main/java/org/apache/iotdb/db/qp/logical/crud/InsertOperator.java]: thanks a lot

**Review by qiaojialin — APPROVED** [review]

**Pull request merged**